### DISCUSSION TITLE
Pythonでのインストール手順(Jetsonを使った場合)

### DISCUSSION THREAD

**Discussion by Takamatsu-Matrix** [ponoor/step-series-support#13] [General]
いろいろ教えていただきながら、インストールしましたが、うまく出来たり出来なかったりで、いろいろ難儀しました。
それで、何台かインストールした中で、うまくいったパターンを記載しておきます。

1 sudo apt update
2 sudo apt upgrade
3 sudo apt-get update
4 sudo apt-get install python3-pip
5 pip3 install -U pip
6 sudo apt install build-essential libbz2-dev libdb-dev libreadline-dev libffi-dev libgdbm-dev liblzma-dev libncursesw5-dev libsqlite3-dev libssl-dev zlib1g-dev uuid-dev tk-dev
7 pythonダウンロード
8 cd Downloads
9 tar Jxfv Python-3.10.4.tar.xz
10 cd Python-3.10.4
11 ./configure
12 make
13 sudo make altinstall
14 cd ~
15 python3.10 -m venv testenv
16 git clone https://github.com/ponoor/python-step-series.git
17 export PATH="$PATH:home/(username)/Desktop/step-series"
18 cd python-step-series
19 git checkout documentation
20 cd ..
21 . testenv-pss/bin/activate
22 python3 -m pip install -e .→これが上手く動かなかった。
23 /home/(username)/testenv/bin/python3 -m pip install --upgrade pip
24 python3 -m pip install -e .→これが上手く動かなかった。
25 python3.10 -m pip install -e .→これが上手く動かなかった。
26 python3.10 -m pip install python-step-series

うまくいかなかったところを飛ばしてやってみると入るようです。

**Comment by kanta** [comment]
@Takamatsu-Matrix 貴重な情報ありがとうございます！
PATHを通すところは、ライブラリ作者Julianさんのコメントの僕の読み間違いで、特に行う必要はなさそうでした。現在documentationブランチでドキュメントの日本語訳を進めていまして、この作業が終わりましたらこのブランチはmasterに統合予定とのことでしたので、`git clone` 以降の作業はかなり簡略化できるかと思います。
